### DISCUSSION TITLE
"Merging" beets libraries

### DISCUSSION THREAD

**Discussion by RollingStar** [beetbox/beets#4950] [Q&A]
I have an existing beets library i stopped updating around 2020. I also have a new one i started in 2023. The old db is set up just the way I want- MBIDs for most, asis for some. there are some extra files included in the library with plugins. (My new beets install also handles extra files). files do not overlap between the two.

What's the best way to bring the old library into the new one? import asis, which would keep the mbids where they exist in the tags? doing a merge in sqlite seems like a bad idea.

**Comment by arsaboo** [comment]
If the old files were imported and tagged correctly, I would just reimport them. If the files are tagged correctly, the search is very fast. 

Even SQL merge is not a bad idea. I would follow it up with a regular import. If you go down that route, I would add the MBIDs and the paths and then import.
